undertow/src/main/resources/schema/wildfly-undertow_3_0.xsd

```diff
@@ -535,5 +535,6 @@
         <xs:attribute name="request-queue-size" type="xs:int" use="optional" />
         <xs:attribute name="security-realm" type="xs:string" use="optional" />
         <xs:attribute name="use-alias" type="xs:string" use="optional" default="false" />
+        <xs:attribute name="enable-http2" type="xs:string" use="optional" default="false" />
```

Comment:
AFAIK the norm is to use the correct type and ignore the expression aspect. Most attributes support expressions, and I don't think being able to validate a document when everything is just xs:string is particularly meaningful. It doesn't prove much beyond valid structure, and unless every attribute with an expression follows this convention the document won't validate anyway.

I'm not really commenting on this PR here, as it looks like this xsd already uses xs:string everywhere. Just joining a conversation. :)